### ISSUE TITLE
Seeking for suggestions on reproducing the experiment results.

### ISSUE THREAD

**Issue by JingfengYang** [jacobandreas/geca#3]
Hi, I have some difficulty in reproducing the reported results in the paper. For example, when I directly run geca/exp/semparse_geo_sql/query_eval_retrieval/run.sh, the best exact match accuracy is 0.36 (reported result for sql-query+GECA is 0.49). When I run geca/exp/semparse_geo_sql/question_eval_retrieval/run.sh, the best exact match accuracy is 0.62 (reported result for sql-question+GECA is 0.68). I found that much augmented data in compose.i.json are not very good. Do you have any suggestions that I can improve the performance? Do I need to tune hyper-parameters in run.sh?

By the way, when I run baseline models using geca/exp/semparse_geo_sql/query_eval_baseline/run.sh and Jia's scripts, the best exact-match result is 0.67 after running with 10 random seeds, which is similar to the results in your paper: sql-question-seq2seq 0.68. Are reported result in your paper the best performance or the average performance after running with 10 random seeds?

Thanks for your help.

**Comment by jacobandreas** [comment]
Hey, sorry this isn't working out of the box!

Possibilities are (1) I forgot to check in the file with the final
hyperparameters, (2) some of the default parameters changed and the scripts
weren't updated to reflect it, or (3) something in the code itself broke
downstream of these experiments. I'll be able to take a look at this in the
next week or two, but if the issue is (2) or (3) it might be resolved by
rolling back to 6be7628.

I'll let you know what I figure out.

**Comment by JingfengYang** [comment]
> Hey, sorry this isn't working out of the box! Possibilities are (1) I forgot to check in the file with the final hyperparameters, (2) some of the default parameters changed and the scripts weren't updated to reflect it, or (3) something in the code itself broke downstream of these experiments. I'll be able to take a look at this in the next week or two, but if the issue is (2) or (3) it might be resolved by rolling back to [6be7628](https://github.com/jacobandreas/geca/commit/6be7628dcbf7d6f8a5307fd9b5e3d3f7af06d9c5). I'll let you know what I figure out.
> […](#)

Thanks so much!

**Comment by jacobandreas** [comment]
OK, it took a little longer than two weeks, but I've verified that rolling back to the commit I mentioned does appear to reproduce the GeoQuery results (I just did a run of sql/query/retrieval and got a test acc of 49). I also put together a minimal example with the Lake, Linzen & Baroni colors dataset, which works on the old version but not the head of the repo. Will try to track down where the bug was introduced.

**Comment by jacobandreas** [comment]
actually, can you say more about how you're running the code? I just tested with a current version of master and was able to get 54% on SQL / query / GECA so I'm not sure what the issue is.

**Comment by JingfengYang** [comment]
Hi, I cloned your code and ran it again today. It worked well this time. It can actually got 54% on SQL / query / GECA.  I think it was due to some bugs on my side when I ran it previously. Sorry for bothering you. And thanks for adding experiments of  Lake, Linzen & Baroni colors dataset .

**Comment by jacobandreas** [comment]
No problem---always good to double-check! Let me know if you run into any
other issues.
